chord_metadata_service/mohpackets/docs/schema_manual.yml

```
@@ -4895,6 +4385,20 @@ components:
       - Wide Local Excision
       title: SurgeryTypeEnum
       type: string
+    SurgeryReferenceDatabaseEnum:
+      enum:
+      - SNOMED
+      - NCIt
+      - UMLS
+      title: SurgeryReferenceDatabaseEnum
+      type: string
+    SystemicTherapyTypeEnum:
+      enum:
+      - Chemotherapy
+      - Hormone Therapy
```

Spreadsheet says this should have lowercase on the therapy
```suggestion 
      - Hormone therapy
```